_posts/2018-03-15-this-week-in-rustfest-2.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: "These weeks in RustFest 2: Diversity Tickets & CFP Last Call"
+authors:
+  - hoverbear
+---
+
+# CFP Last Call
+
+<div style="text-align: center">
+  <strong><a href="https://cfp.rustfest.eu/events/rustfest-paris">March 17th (tomorrow!) is the CFP deadline! No extensions!</a></strong>
+  <br>
+  <iframe src="https://giphy.com/embed/l3vRkWqpxQW3wm6ek" width="480" height="360" frameBorder="0" class="giphy-embed" allowFullScreen></iframe>
+</div>
```

Comment:
I'd say no fast-moving gif as the very first thing please (also if we _do_ add a gif, why as an iframe? Why not the direct link?)